### DISCUSSION TITLE
Remove support for Ehcache

### DISCUSSION THREAD

**Discussion by Luke-Sikina** [cBioPortal/cbioportal#8674] [Ideas]
Currently we have three caching solutions:
- No cache
- Redis cache
- Ehcache

To the best of my knowledge, no one uses ehcache aside from the MSK private portal. Generally, a feature that is not used is a feature that accumulates bugs. I propose that when we switch the MSK portal to Redis, we remove the Ehcache option. This has a few advantages:
- Wiring: if we remove Ehcache, we can just always wire the redis cache, and disable it at runtime. This allows us to only create one docker container that serves all our caching options
- Simplicity: no confusion over what cache to use

**Comment by jjgao** [comment]
@cBioPortal/backend
